metadata-ws/src/main/resources/application.properties

```diff
@@ -46,3 +46,7 @@ swagger.apiinfo.license=Apache 2.0
 swagger.apiinfo.licenseUrl=http://www.apache.org/licenses/LICENSE-2.0
 swagger.apiinfo.version=@project.version@
 swagger.apiinfo.termsOfServiceUrl=
+
+security.enabled=true
+security.oauth2.resource.userInfoUri=http://ampt2d-gait.windows.ebi.ac.uk:8880/ega/ampt2d/auth/realms/Ampt2d/protocol/openid-connect/userinfo
```

Comment:
Is this supposed to be hard-coded? Maybe this should be stored as an external `@parameter@` instead?

Reply:
Well, my understanding is this works for AMP-T2D. I will change this to @metadata-userInfoUri@